tests/src/scope/scope.ts

```diff
@@ -0,0 +1,61 @@
+import assert from "assert";
+import { parseForESLint } from "../../../src";
+import { getScopeFromNode } from "../../../src/scope";
```

Comment:
Thank you for this PR.

Can you replace the test with ESLint's `context.getScope` instead of `getScopeFromNode`? `getScopeFromNode` is not used in the actual rule implementation.